### ISSUE TITLE
The application crashed after entering the Beneficiary

### ISSUE THREAD

**Issue by danicricco** [benetech/ODKSurveyServices#16]
After filling the form of "Beneficiary Information", the application crashed with an unexpected error. It is blocking me to further test the application.
After crashing the application asked to send feedback. I have included the issue number as feedback. I hope this will help matching this ticket description with the stacktrace.

## Settings is pointing to the QA server

- URL: https://povertystoplight-ws-qa.benetech.org
- Office Id: testing

## Device input
- Android 7.0
- Samsung Galaxy S7 edge

**Comment by danicricco** [comment]
The problem could be avoided by:

1. Deleting the app
2. Deleting the folder "opendatakit"
3. Fresh install of the app

**Comment by johnhbenetech** [comment]
Dani,

Do you remember  if you had a previous install of the app before installing the playstore version? I wonder if an existing 'opendatakit' directory was present on the device during install.

We have a change related to the beneficiary/reporter information being populated  that may help prevent this issue in the future. I'll push it to the playstore this morning after I test it internally.

Thanks

**Comment by danicricco** [comment]
Yes, Indeed, I had the previous version. I removed it before installing, but didn't delete the opendatakit folder.

I think that we should consider this report, since it might be difficult for and end-user to delete the opendatakit folder.

**Comment by johnhbenetech** [comment]
The hope is that the end-user won't need to delete the opendatakit folder ever. The old version on your device was incompatible with the first official alpha release. Going forward we dont plan on the need to delete the opendatakit folder once we fix a few of the existing bugs
